### PR TITLE
remove job from image_export

### PR DESCRIPTION
**Pull request by LK4D4**
Closes #12596 
I added `Flushed` logic that returns json error if `200 OK` was already written.

### CODE REVIEW

File: graph/service.go [hunk 1]
```diff
@@ -11,7 +11,6 @@ import (
 func (s *TagStore) Install(eng *engine.Engine) error {
 	for name, handler := range map[string]engine.Handler{
 		"image_inspect": s.CmdLookup,
-		"image_export":  s.CmdImageExport,
 		"viz":           s.CmdViz,
 		"load":          s.CmdLoad,
```
Comment (runcom) [inline graph/service.go]: mmm why load and push are here if master haven't them (https://github.com/docker/docker/blob/master/graph/service.go)

Comment (LK4D4) [inline graph/service.go]: Rebase needed.

**Comment by duglin** [comment]
LGTM once janky is done

**Comment by jfrazelle** [comment]
LGTM

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**